scripts/utils/create_package_template.py

```diff
@@ -27,7 +27,7 @@
     "Hex Editors",
     "Java",
     "Javascript",
-    "Net",
+    "Networking",
```

Comment:
are these the final categories? They are different from https://github.com/mandiant/VM-Packages/issues/38#issuecomment-1313874710

As already mentioned in this PR, modifying the categories also needs to update our current GH actions and the packages that are using the old categories (and maybe somewhere else?).

Reply:
These are not final, no. Will handle that as part of #38.

Reply:
Updated them now per the last comment in #38.